Source/WebCore/platform/graphics/skia/GraphicsContextSkia.cpp

```diff
@@ -284,6 +289,42 @@ void GraphicsContextSkia::strokePath(const Path& path)
     canvas().drawPath(*path.platformPath(), createStrokePaint());
 }
 
+sk_sp<SkImageFilter> GraphicsContextSkia::createDropShadowFilter(ShadowStyle shadowStyle) const
+{
+    // FIXME: this does not handle state.shadowsIgnoreTransforms()
+
+    if (!hasDropShadow())
+        return nullptr;
+
+    const auto shadow = dropShadow();
+    ASSERT(shadow);
+
+    FloatSize offset = shadow->offset;
```

Comment:
We don't need to copy the offset, so either use shadow->offset directly or get a const reference.